Python/bytecodes.c

```diff
@@ -395,7 +397,7 @@ dummy_func(
         }
 
         op(_REPLACE_WITH_TRUE, (value -- res)) {
-            Py_DECREF(value);
+            DECREF_INPUTS();
```

Comment:
The code for `DECREF_INPUTS` doesn't look right to me:

```
            owner_stackref = stack_pointer[-1];
            owner = PyStackRef_Get(owner_stackref);
...
            value_stackref = PyStackRef_StealRef(owner);
            {
                (void)value;
                PyStackRef_DECREF(value_stackref);
                res = Py_True;
            }
```

This loses the information from the `PyStackRef_Get`. We should be doing:

```
PyStackRef_DECREF(stack_pointer[-1]);
```

Reply:
Hmm in that case, we should pass through owner_stackref to value_stackref then.